stdlib/unittest/mock.pyi

```diff
@@ -1,5 +1,6 @@
 import sys
 from _typeshed import Self
+from contextlib import AbstractContextManager
```

Comment:
```suggestion
from contextlib import _GeneratorContextManager
```

It's best to be as precise as possible when annotating return types, and this is the actual return type of functions decorated with `@contextmanager` :)

Reply:
Cool, didn't know about it. However, I don't see it being used anywhere in the stubs of stdlib. Is that on purpose?

Reply:
I don't _think_ it's on purpose 🤷

Reply:
OK, done :)

Reply:
Thanks!